src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
```

Comment:
we need to import `calculate.js` in here right?

Pathwright 👇 
![image](https://user-images.githubusercontent.com/31897434/81283722-fd53c680-9054-11ea-841c-6896bf6d459f.png)